### DISCUSSION TITLE
Can't delete external models

### DISCUSSION THREAD

**Discussion by 10067** [open-webui/open-webui#6726] [Issues]
Discussion https://github.com/open-webui/open-webui/discussions/7054

Go to 

```
Admin Panel
    ➡️ Settings
        ➡️Connections
             ➡️ Manage Ollama API connections
                 ➡️ Manage (🔧 icon) 
```
![image](https://github.com/user-attachments/assets/1aed8eb8-bc90-4b35-af70-fc8e22711152)

**Comment by 2026DATUS** [comment]
it is impossible to delete now upload and delete models in Openweb version 0.4.2, how do I do it now ?

**Comment by fiveloop** [comment]
Same here

**Comment by qua1121** [comment]
I have the same problem and I cannot find the correct way to deal with this situation from documents ( https://docs.openwebui.com/features/workspace/models/ )

**Comment by sirweazel** [comment]
Same... But I'm currently on mobile and don't see any way to manage (delete or update) models.
